src/com/palmergames/bukkit/towny/db/TownySQLSource.java

```diff
@@ -928,6 +928,17 @@ public boolean loadTown(Town town) {
                 } catch (SQLException e) {
                 }
 
+                try {
+                    line = rs.getString("registered");
+                    if (line != null){
```

Comment:
Not sure if this will come up null if we're setting it to "undefined", same for the below loadNation

Reply:
No longer setting it to undefined, its being set to 0 now